docs/redoc/master/openapi.json

```diff
@@ -6191,9 +6191,7 @@
             "type": "boolean"
           },
           "rescore": {
-            "description": "If true, use original vectors to re-score top-k results. Might require more time in case if original vectors are stored on disk. Default is false.",
-            "default": false,
-            "type": "boolean"
+            "$ref": "#/components/schemas/Rescoring"
```

Comment:
this is not backward compatible.

Reply:
okay, will revert to `Option<bool>` and will add more text comments about `None` behaviuor